amaranth/lib/wiring.py

```diff
@@ -570,6 +570,9 @@ def __getattr__(self, name):
         value = getattr(self.__unflipped, name)
         if inspect.ismethod(value):
             return types.MethodType(value.__func__, self)
+        elif name in self.__unflipped.signature.members and \
```

Comment:
it's usually prefered to put long conditions in parentheses instead of using backslash continuation

Reply:
Should I make a new PR to add `flake8` to `dev-dependencies`, a `.flake8` file (since `flake8` does not support `pyproject.toml` and it is apparently [nontrivial](https://github.com/PyCQA/flake8/issues/234) to add) and: https://github.com/planetlabs/flake8-continuation?

Reply:
No flake8.